manifests/c/CABS/ScanPrix/6.15.0.0/CABS.ScanPrix.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: CABS.ScanPrix
+PackageVersion: 6.15.0.0
+InstallerType: nullsoft
+Installers:
+- Architecture: x86
+  InstallerUrl: https://www.scanprix.de/Setup/Setup_ScanPrix.exe
+  InstallerSha256: FF3DBF127207A8B72AEDF619FCB59057D7B9244E37844417CB4ADF01C8D501E2

```

```suggestion
  InstallerSha256: FF3DBF127207A8B72AEDF619FCB59057D7B9244E37844417CB4ADF01C8D501E2
  ProductCode: ScanPrix-Programme
  AppsAndFeaturesEntries:
  - DisplayName: CABS-Deinstaller
    ProductCode: ScanPrix-Programme
```
Apps and features entries are needed since the DisplayName does not match the PackageName

---

why should i replace this line? Both are identically.